plugins/env/app/controllers/environment_variable_groups_controller.rb

```diff
@@ -42,14 +42,17 @@ def destroy
   end
 
   def preview
-    @groups =
-      if params[:group_id]
-        @group = EnvironmentVariableGroup.find(params[:group_id])
-        SamsonEnv.env_groups(Project.new(environment_variable_groups: [@group]), DeployGroup.all, preview: true)
-      else
-        @project = Project.find(params[:project_id])
-        SamsonEnv.env_groups(@project, DeployGroup.all, preview: true)
-      end
+    @deploy_group = DeployGroup.all
+    @deploy_group = @deploy_group.where(permalink: params[:scope]) if params[:scope].present?
```

Comment:
don't name it scope ... and add a test :)

Reply:
what should it be named then? it says scope in the UI? and permalink seems anything but descriptive, for something that's not really a link.